src/inmanta/protocol/common.py

```diff
@@ -65,9 +66,9 @@ class ArgOption(object):
 
     def __init__(
         self,
+        getter: Callable[[Any, Dict[str, str]], Coroutine[Any, Any, Any]],
```

Comment:
add a comment to indicate that we do Any instead of generic type, because we don't do typecheck between hander and method at all.